debian/changelog

```diff
@@ -1,6 +1,6 @@
-nicotine (1.4.3-2) UNRELEASED; urgency=medium
+nicotine (1.4.3-2) eoan; urgency=medium
```

Comment:
Thanks @mathiascode. Since `focal` is stable now, you can update this. Although the PPA build recipe automatically replaces this with the release it's building for anyways.